_docs/Backend/gettingstarted.md

```diff
@@ -16,7 +16,11 @@ _Docker makes this project easier to work with. But if you don’t know [how to
 3. [Install and run a local mongo client](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/)
 4. _cp .env.example .env_
 
-In env, make sure RACCOON_REDIS_URL=localhost and MONGO_HOST_TEST=mongodb://localhost/express-mongoose-es6-rest-api-development-test
+In env file 
+1. set _RACCOON_REDIS_URL_=localhost. 
+2. set _MONGO_HOST_TEST=mongodb://localhost/express-mongoose-es6-rest-api-development-test_ and _NODE_ENV_=test if setting up _TEST_ environemnt.
```

Comment:
You don't need to set env to test. This only get set when running tests.

Reply:
Got it. Thanks for the clarification.

Reply:
Np! Did you need to set it for test for any reason? Maybe we are missing docs on how to do testing.

Reply:
@TheHollidayInn I did not need to set it for test. I was just following the initial documentation and  which said to  set MONGO_HOST_TEST to the local mongo instance and then run ```npm start``` instead of ```npm test```.  This combination did not work and I was not aware that env gets set to test automatically when running test. So I ended up manually updating the NODE_ENV to test to get the backend server up and running